src/containers/Airdrop/AirdropGiftTracker/components/Claimmodal/ClaimModal.tsx

```diff
@@ -82,9 +93,14 @@ export default function ClaimModal({ onSubmit, onClose }: ClaimModalProps) {
                             <XLogoIcon />
                         </XLogo>
                     </ClaimButton>
-                </ActionWrapper>
-                <ActionWrapper>
-                    <TextButton onClick={onClose}>{t('doLater')}</TextButton>
+
+                    {!allowTelemetry && (
+                        <FinePrint
+                            dangerouslySetInnerHTML={{
```

Comment:
can we use the `Trans` component for this? :D _dangerously_ always makes me nervous:p

Reply:
too dangerous! done!